<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectrumWindow</class>
 <widget class="QMainWindow" name="SpectrumWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spectrum</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="signalButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>O1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout_4" rowstretch="1,0,0,0">
          <item row="3" column="2">
           <widget class="QLabel" name="o1_theta_percent">
            <property name="text">
             <string>0.00%</string>
            </property>
           </widget>
          </item>
          <item row="3" column="5">
           <widget class="QLabel" name="o1_gamma_percent">
            <property name="text">
             <string>0.00%</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="o1_theta_raw">
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QLabel" name="o1_beta_percent">
            <property name="text">
             <string>0.00%</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>Percent</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>Alpha</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>Gamma</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="o1_alpha_percent">
            <property name="text">
             <string>0.00%</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QLabel" name="o1_beta_raw">
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>Theta</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>Beta</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_28">
            <property name="text">
             <string>Delta</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="o1_delta_percent">
            <property name="text">
             <string>0.00%</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>Raw</string>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QLabel" name="o1_gamma_raw">
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="o1_alpha_raw">
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="o1_delta_raw">
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="6">
           <layout class="QVBoxLayout" name="o1_graphLayout"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>O2</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,0,0,0">
          <item row="3" column="2">
           <widget class="QLabel" name="o2_theta_percent">
            <property name="text">
             <string>0.00%</string>
            </property>
           </widget>
          </item>
          <item row="3" column="5">
           <widget class="QLabel" name="o2_gamma_percent">
            <property name="text">
             <string>0.00%</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="o2_theta_raw">
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QLabel" name="o2_beta_percent">
            <property name="text">
             <string>0.00%</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Percent</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Alpha</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Gamma</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="o2_alpha_percent">
            <property name="text">
             <string>0.00%</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QLabel" name="o2_beta_raw">
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Theta</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Beta</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Delta</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="o2_delta_percent">
            <property name="text">
             <string>0.00%</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>Raw</string>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QLabel" name="o2_gamma_raw">
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="o2_alpha_raw">
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="o2_delta_raw">
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="6">
           <layout class="QVBoxLayout" name="o2_graphLayout"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>T3</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,0,0,0">
          <item row="1" column="2">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Theta</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QLabel" name="t3_beta_raw">
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="t3_delta_percent">
            <property name="text">
             <string>0.00%</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Raw</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QLabel" name="t3_beta_percent">
            <property name="text">
             <string>0.00%</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Gamma</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Beta</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="t3_delta_raw">
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="6">
           <layout class="QVBoxLayout" name="t3_graphLayout"/>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="t3_alpha_raw">
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="t3_theta_percent">
            <property name="text">
             <string>0.00%</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Delta</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="t3_theta_raw">
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="t3_alpha_percent">
            <property name="text">
             <string>0.00%</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Alpha</string>
            </property>
           </widget>
          </item>
          <item row="3" column="5">
           <widget class="QLabel" name="t3_gamma_percent">
            <property name="text">
             <string>0.00%</string>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QLabel" name="t3_gamma_raw">
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Percent</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="t4Tab">
       <attribute name="title">
        <string>T4</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0">
          <item row="3" column="2">
           <widget class="QLabel" name="t4_theta_percent">
            <property name="text">
             <string>0.00%</string>
            </property>
           </widget>
          </item>
          <item row="3" column="5">
           <widget class="QLabel" name="t4_gamma_percent">
            <property name="text">
             <string>0.00%</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="t4_theta_raw">
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QLabel" name="t4_beta_percent">
            <property name="text">
             <string>0.00%</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Percent</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Alpha</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Gamma</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="t4_alpha_percent">
            <property name="text">
             <string>0.00%</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QLabel" name="t4_beta_raw">
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Theta</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Beta</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Delta</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="t4_delta_percent">
            <property name="text">
             <string>0.00%</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Raw</string>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QLabel" name="t4_gamma_raw">
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="t4_alpha_raw">
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="t4_delta_raw">
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="6">
           <layout class="QVBoxLayout" name="t4_graphLayout"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
